> tests passed first try!

I had to open my big mouth didn't I. 😂 It is failing reliably on macos13/pyqt5:

```pytb
=================================== FAILURES ===================================
  __________________________ test_camera_orientation_3d __________________________
  
  make_napari_viewer = <function make_napari_viewer.<locals>.actual_factory at 0x32f039ee0>
  
      def test_camera_orientation_3d(make_napari_viewer):
          """Test that flipping camera orientation in 3D flips volume as expected."""
          viewer = make_napari_viewer()
          viewer.dims.ndisplay = 3
          gradient_z = np.arange(16).reshape((16, 1, 1))
          image = np.ones((16, 16))
          image_3d = gradient_z * image
          _ = viewer.add_image(image_3d)
      
          # We have a 3D image with brightness *increasing with z* (the 0th axis).
          # Therefore, when using a perspective projection, if we point z *away* from
          # us, the bright part of the image will be small, whereas if we point z
          # *towards* us, it will be large. We test the dimension flip by comparing
          # the overall brightness of the image when pointing z in different
          # directions
      
          viewer.camera.perspective = 60
          viewer.camera.orientation = ('away', 'down', 'right')
          sshot_away = viewer.screenshot(canvas_only=True, flash=False)[..., 0]
          viewer.camera.orientation = ('towards', 'down', 'right')
          sshot_towards = viewer.screenshot(canvas_only=True, flash=False)[..., 0]
      
  >       assert np.mean(sshot_towards) > np.mean(sshot_away)
  E       assert np.float64(0.0) > np.float64(0.0)
  E        +  where np.float64(0.0) = <function mean at 0x1048e89b0>(array([[0, 0, 0, ..., 0, 0, 0],\n       [0, 0, 0, ..., 0, 0, 0],\n       [0, 0, 0, ..., 0, 0, 0],\n       ...,\n       [0, 0, 0, ..., 0, 0, 0],\n       [0, 0, 0, ..., 0, 0, 0],\n       [0, 0, 0, ..., 0, 0, 0]], dtype=uint8))
  E        +    where <function mean at 0x1048e89b0> = np.mean
  E        +  and   np.float64(0.0) = <function mean at 0x1048e89b0>(array([[0, 0, 0, ..., 0, 0, 0],\n       [0, 0, 0, ..., 0, 0, 0],\n       [0, 0, 0, ..., 0, 0, 0],\n       ...,\n       [0, 0, 0, ..., 0, 0, 0],\n       [0, 0, 0, ..., 0, 0, 0],\n       [0, 0, 0, ..., 0, 0, 0]], dtype=uint8))
  E        +    where <function mean at 0x1048e89b0> = np.mean
  
  napari/_vispy/_tests/test_vispy_camera.py:203: AssertionError
```

@Czaki any ideas why the canvas would not be drawing correctly here? Since it's not really a supported config (we are not supporting pyqt5 on py3.13 in general, no?), I could just skip/mark as xfail under those conditions?

--- a/napari/_vispy/_tests/test_vispy_camera.py
+++ b/napari/_vispy/_tests/test_vispy_camera.py
@@ -1,4 +1,9 @@
+import os
+import sys
+
 import numpy as np
+import pytest
+from qtpy import PYQT5
 
 
 def test_camera(make_napari_viewer):
@@ -115,6 +120,15 @@ def test_vispy_camera_update_from_model_3D(make_napari_viewer):
     np.testing.assert_almost_equal(viewer.camera.zoom, vispy_camera.zoom)
 
 
+@pytest.mark.xfail(
+    condition=(
+        sys.version_info >= (3, 13)
+        and sys.platform.startswith('darwin')
+        and os.getenv('CI', '0') != '0'
+        and PYQT5
+    ),
+    reason='test fails on this specific CI config for some reason',
+)
 def test_camera_model_update_from_vispy_3D(make_napari_viewer):
     """Test camera model updates from vispy in 3D."""
     viewer = make_napari_viewer()

--- a/napari/_vispy/_tests/test_vispy_camera.py
+++ b/napari/_vispy/_tests/test_vispy_camera.py
@@ -120,15 +120,6 @@ def test_vispy_camera_update_from_model_3D(make_napari_viewer):
     np.testing.assert_almost_equal(viewer.camera.zoom, vispy_camera.zoom)
 
 
-@pytest.mark.xfail(
-    condition=(
-        sys.version_info >= (3, 13)
-        and sys.platform.startswith('darwin')
-        and os.getenv('CI', '0') != '0'
-        and PYQT5
-    ),
-    reason='test fails on this specific CI config for some reason',
-)
 def test_camera_model_update_from_vispy_3D(make_napari_viewer):
     """Test camera model updates from vispy in 3D."""
     viewer = make_napari_viewer()
@@ -192,6 +183,15 @@ def test_camera_orientation_2d(make_napari_viewer):
     assert np.all(avg_col_intensity_grad2 <= 0)
 
 
+@pytest.mark.xfail(
+    condition=(
+        sys.version_info >= (3, 13)
+        and sys.platform.startswith('darwin')
+        and os.getenv('CI', '0') != '0'
+        and PYQT5
+    ),
+    reason='test fails on this specific CI config for some reason',
+)
 def test_camera_orientation_3d(make_napari_viewer):
     """Test that flipping camera orientation in 3D flips volume as expected."""
     viewer = make_napari_viewer()

--- a/napari/components/camera.py
+++ b/napari/components/camera.py
@@ -29,12 +29,12 @@ class DepthAxisOrientation(StringEnum):
     TOWARDS = auto()
 
 
-DEFAULT_ORIENTATION = ('towards', 'down', 'right')
 DEFAULT_ORIENTATION_TYPED = (
     DepthAxisOrientation.TOWARDS,
     VerticalAxisOrientation.DOWN,
     HorizontalAxisOrientation.RIGHT,
 )
+DEFAULT_ORIENTATION = tuple(map(str, DEFAULT_ORIENTATION_TYPED))
 
 
 class Camera(EventedModel):

